extensions/mssql/package.json

```diff
@@ -1052,10 +1053,17 @@
   },
   "devDependencies": {
     "@types/bytes": "^3.0.0",
+    "@types/chai": "^4.2.11",
     "@types/kerberos": "^1.1.0",
+    "@types/mocha": "^7.0.2",
     "@types/request": "^2.48.2",
     "@types/request-promise": "^4.1.44",
+    "@types/sinon": "^7.5.2",
     "@types/stream-meter": "^0.0.22",
-    "@types/through2": "^2.0.34"
+    "@types/through2": "^2.0.34",
+    "chai": "^4.2.0",
+    "mocha": "^7.0.2",
```

Comment:
Tests should be added using vscodetestcover like https://github.com/microsoft/azuredatastudio/blob/master/extensions/cms/package.json#L635

And then the coverage config file added https://github.com/microsoft/azuredatastudio/blob/master/extensions/cms/coverConfig.json

And also added to our test scripts

https://github.com/microsoft/azuredatastudio/blob/master/scripts/test-extensions-unit.bat
https://github.com/microsoft/azuredatastudio/blob/master/scripts/test-extensions-unit.sh

Reply:
Added the requested files/updates.